doc/nrf/protocols/matter/overview/dev_model.rst

```diff
@@ -39,6 +39,8 @@ The following table lists Matter versions supported in the |NCS|, with a brief o
 +-----------------+----------------------------------------------+---------------------+
 | Matter version  | Overview of changes                          | Release date        |
 +=================+==============================================+=====================+
+| 1.1.0           |                                              |                     |
```

Comment:
As we discussed F2F, here is a list of changes:

- Introduced support for persistent subscriptions
- Added certification tests verifying usage of multiple credentials types at the same time, such as pin codes, RFID or fingerprint, for a Door Lock device type (cluster implementation was present in Matter 1.0, but it was not verified).
- Introduced significant editorial changes and clarifications in the specification, improved alignment of the Matter specification with the Matter SDK , and added several bug fixes.